fe/fe-core/src/main/java/com/starrocks/connector/hive/CacheUpdateProcessor.java

```diff
@@ -160,6 +249,35 @@ private void refreshRemoteFiles(String tableLocation, Operator operator, List<St
         refreshRemoteFiles(tableLocation, operator, existPaths, onlyCachedPartitions, executor);
     }
 
+    private void refreshRemoteFilesBackground(String tableLocation, List<String> updatePaths,
```

Comment:
I think `refreshRemoteFilesBackground` and `refreshRemoteFiles` have too much duplicate code. It's better to integrate them.